config/database.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    // TODO - replace 'app_db' with shorthand app name once that is decided
+    url : 'mongodb://localhost/app_db'  // using default port (27017)
+}
```

Comment:
and then move this to dev.config.js
and we maybe throw a secret key on there just placeholder secret: 'yourSecret' or whatever